src/App.tsx

```diff
@@ -1,9 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setFilteredMovies] = useState(moviesFromServer);
```

Comment:
you don't need it, keep only common `filteredMovies` (or rename it to `visibleMovies`) const